test/suite/test_backup01.py

```diff
@@ -31,15 +31,15 @@
 import shutil
 import string
 import time
-from suite_subprocess import suite_subprocess
-import wiredtiger, wttest
+from wtbackup import test_backup_base
```

Comment:
The internal class name should also change name from `test_backup_base`. No only for cleanliness, but also the test harness library we use for python attributes special meaning to things (files and functions) prepended with `test_` and while it may work now, it could lead to subtle bugs if that library changes.

Reply:
fair, will call it **backup_base**